app/helpers/safe-markup.js

```diff
@@ -1,6 +1,12 @@
 import Ember from 'ember';
 
-export function safeMarkup(params/*, hash*/) {
+/**
+  * safeMarkup helper. Returns a string that will not be HTML escaped by Handlebars.
+  *
+  * @method safeMarkup
+  * @param {string} string
+  * @return {Object} Returns Handlebars.SafeString
+  */export function safeMarkup(params/*, hash*/) {
```

Comment:
Thanks for adding a docstring! Can you confirm that this actually shows up in the generated docs when you run `yarn run docs`?

(you don't need to commit the changes, but please verify they show up)

1. Can the export statement please be on a separate line?
2. For convention sake, I've been trying to write the helper name as it would appear when used (eg `safe-markup`). Not sure if this causes problems for yuidoc- thoughts?